app/main.py

```diff
@@ -16,4 +16,26 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins,
+]
+
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for char in variables:
```

Comment:
The loop variable `char` is misleadingly named, as it suggests that it iterates over characters. Consider renaming it to something more descriptive, like `var` or `variable`, to better reflect its purpose.